torch/_dynamo/output_graph.py

```diff
@@ -2191,13 +2192,18 @@ def lift_tracked_freevar_to_input(self, proxy):
         # If that is the case, just return the already lifted Proxy.
         if proxy in self.lifted_freevars:
             return self.lifted_freevars[proxy]
+
+        # We first lift proxy to parent's graph then lift to current grpah's input
+        # so that when we bind symints of the sizes in current graph, those symints
+        # would already be lifted as inputs to parent graph.
+        if proxy.tracer != self.parent:
+            self.parent.lift_tracked_freevar_to_input(proxy)
```

Comment:
Why don't we need the `self.parent is None` condition anymore?

Reply:
We have an assertion in L2188 that asserts self.parent is not None